front/lib/resources/group_resource.ts

```diff
@@ -652,6 +652,22 @@ export class GroupResource extends BaseResource<GroupModel> {
     workspace: LightWorkspaceType,
     transaction?: Transaction
   ) {
+    const groups = await this.model.findAll({
```

Comment:
You can do: 

```
 await GroupVaultModel.destroy({
      where: {
        workspaceId: workspace.id,
      },
      transaction,
    });
```

Reply:
Oh wait 😄

Reply:
🙈

Reply:
That's what we do for GroupMembership cause it was linked to the workspace directly. 
Not sure why we decided to do it for GroupMembership and not for GroupVault, cause it's really convenient!

Reply:
Want me to edit the `GroupVaultModel` to add a foreign key on `Workspace`?

Reply:
Otherwise I am afraid we need to retrieve vaultsIds first